src/core/ext/transport/chttp2/transport/call_tracer_wrapper.h

```diff
@@ -0,0 +1,72 @@
+//
+//
+// Copyright 2024 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//
+
+#ifndef GRPC_SRC_CORE_EXT_TRANSPORT_CHTTP2_TRANSPORT_CALL_TRACER_WRAPPER_H
+#define GRPC_SRC_CORE_EXT_TRANSPORT_CHTTP2_TRANSPORT_CALL_TRACER_WRAPPER_H
+
+#include "src/core/lib/transport/transport.h"
+#include "src/core/telemetry/call_tracer.h"
+
+struct grpc_chttp2_stream;
+
```

Comment:
Forward declaration because grpc_chttp2_stream* is a data member in Chttp2CallTracerWrapper .
Did not include internal.h in this file to avoid Circular dependency because internal.h has grpc_chttp2_stream which has a Chttp2CallTracerWrapper object.